static/js/whyis_vue/components/add-attribute.vue

```diff
@@ -339,15 +339,16 @@ export default Vue.component('add-attribute', {
         },
 
         async getAttributeList (query) {
+	    var combinedList = [];
             const [rdfsProperty, owlDatatypeProperty] = await axios.all([
                 axios.get(
-                `/?term=*${query}*&view=resolve&type=http://www.w3.org/1999/02/22-rdf-syntax-ns%23Property`),
```

Comment:
So I think the issue here is that we aren't guaranteed to be on the root of the server. I'm surprised that the new one is working too. You need to prepend `ROOT_URL`, which is available in the root element of Vue. You're going to need to do this regularly so that we are using the correct URL regardless of the configuration.